src/components/__tests__/Subthoughts.ts

```diff
@@ -593,3 +594,53 @@ describe('expand thoughts', () => {
     expect(thoughtB()).toHaveLength(1)
   })
 })
+
+describe('multi-column mode', () => {
+  it('Multi-column mode must not be active on single nested thought', () => {
+    // import thoughts
+    store.dispatch([
+      importText({
+        text: `- a
+        - =view
+          - Table
+        - b
+          - c
+            - e`,
+      }),
+    ])
+
+    const thoughtB = screen.getAllByText('b').filter(element => element.classList.contains('editable'))[0]
+
+    expect(thoughtB.closest('.is-multi-column')).toBeFalsy()
+  })
+
+  it('Multi-column mode must be active on muliple levels of nested thought', () => {
+    // import thoughts
+    store.dispatch([
+      importText({
+        text: `- Fruit
+        - =view
+          - Table
+        - Apple
+          - Color
+            - Red
+          - Type
+            - Seed
+        - Banana
+          - Color
+            - Yellow
+          - Type
+            - Tropical
+        - Tangerine
+          - Color
+            - Orange
+          - Type
+            - Citrus`,
+      }),
+    ])
+
+    const thoughtRed = screen.getAllByText('Color').filter(element => element.classList.contains('editable'))[0]
+
+    expect(thoughtRed.closest('.is-multi-column')).toBeTruthy()
```

Comment:
This is a little loose, but it's okay. It would be better to test that `is-multi-column` is set on the correct node, rather than just any ancestor.

Reply:
I thought given the constraints in the input, it was not necessary to test it.

Reply:
That's fair.